src/ts/eventdispatcher.ts

```diff
@@ -227,4 +266,9 @@ class RateLimitedEventListenerWrapper<Sender, Args> extends EventListenerWrapper
     // Fire the internal rate-limiting listener instead of the external event listener
     this.rateLimitingEventListener(sender, args);
   }
-}
\ No newline at end of file
+
+  clear(): void {
```

Comment:
This must also be called when unsubscribing an event handler. Then we have to explicitely unsubscribe all event handlers subscribed with `subscribeRateLimited` to avoid leaks.